common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -2,12 +2,23 @@
 
 from collections import namedtuple
 from stevedore.extension import ExtensionManager
+from xblock.fields import List
 
 # We use ``id`` in this file as the IDs of our Groups and UserPartitions,
 # which Pylint disapproves of.
 # pylint: disable=redefined-builtin
 
 
+class UserPartitionList(List):
+    """Special List class for listing UserPartitions"""
```

Comment:
Sigh, you are correct. I just hate to take up the space (and am generally grumpy about having to write documentation lines for things that are self-documenting, as self-documenting should be the goal).

But I suppose I will change all my single-line documentation in this PR. :(